pkg/api/meta/meta.go

```diff
@@ -506,8 +548,20 @@ func (a genericAccessor) SetAnnotations(annotations map[string]string) {
 	*a.annotations = annotations
 }
 
-func (a genericAccessor) GetOwnerReferences() []unversioned.ObjectReference {
-	return a.ownerReferences
+func (a genericAccessor) GetOwnerReferences() []OwnerRefereceAccessor {
```

Comment:
Append instead of insert is fine with me.

But I really think `GetOwnerReference(i int) OwnerRefereceAccessor` is much better than something that returns an array-- my way it's possible to mutate the reference.